queries/CreateAppUser.js

```diff
@@ -10,25 +10,24 @@
 
 module.exports = function (req, appUser, tenant) {
    return new Promise((resolve, reject) => {
-      let tenantDB = "`appbuilder-admin`";
+      let tenantDB = "appbuilder-admin";
```

Comment:
OK, really with the additional check below, we could leave this 
`let tenantDB;`
and not pre add the `appbuilder-admin`